src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -8017,4 +8017,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_IteratorRefLikeElementType" xml:space="preserve">
     <value>Element type of an iterator may not be a ref struct or a type parameter allowing ref structs</value>
   </data>
+  <data name="WRN_UnscopedRefAttributeOldRules" xml:space="preserve">
+    <value>UnscopedRefAttribute does not have any effect in C# 10 or earlier with net6.0 or earlier.</value>
```

Comment:
Not following the `net6.0` part. The attribute should have effect if lang version is 11 or higher. net6.0 becomes relevant because at net7.0 or higher the new rules take effect irrespective of language version.

Reply:
The "updated" ref safety rules apply either in C# 11 or when referencing net7.0. The [speclet](https://github.com/dotnet/csharplang/blob/85612a1abd8b5c66c7c7be20814c79b89ed98808/proposals/csharp-11.0/low-level-struct-improvements.md#refsafetyrulesattribute) says:

> If the module containing the method declaration references System.Runtime { ver: 7.0 }, the method call is analyzed with C#11 rules. This rule is a temporary mitigation for modules compiled with earlier previews of C#11 / .NET 7 and will be removed later.

Reply:
> The "updated" ref safety rules apply either in C# 11 or when referencing net7.0. 

Agreed. But the error says 

> UnscopedRefAttribute does not have any effect in C# 10 or earlier with net6.0 or earlier.

Given the `or` this can be read as 

> UnscopedRefAttribute does not have any with net6.0 or earlier.

I get what the message is trying to say but not sure the wording is 100% clear. Think it would be better if we turned this into a positive declaration

> UnscopedRefAttribute is only valid in C# 11 or later or when targeting net7.0 or later.

Reply:
I see, I will update it, thanks.